modules/ossm-migrating-done-network-policies.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/checklists/ossm-migrating-network-policies.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-migrating-network-policies-security_{context}"]
+= Migrating network policies after migrating deployment and workloads
+
+If you did not recreate your network policies before you migrated your deployment and workloads, you can recreate your network policies now.
+
+.Prerequisites
+
+* You have migrated your deployment.
+* You have migrated your workloads.
+
+[NOTE]
+====
+Use a label scoped specifically to your mesh that you can reuse for discovery selectors.
+====
+
+.Procedure
+
+. Recreate necessary network policies in the new {SMProduct} 3 control plane namespace.
+
+. Recreate network policies for each namespace that was part of the {SMProduct} 2 mesh.
+
+. Update labels.
++
+.. Update corresponding network policy selectors to match the new labels.
```

Comment:
Can this be collapsed into one step? Or should the `..` be bumped to `.`?
![image](https://github.com/user-attachments/assets/4d86bad7-19b0-4538-ab69-12860c58b327)